mmv1/third_party/terraform/transport/config.go.tmpl

```diff
@@ -1358,11 +1358,34 @@ func (c *Config) GetCredentials(clientScopes []string, initialCredentialsOnly bo
 	var creds *googleoauth.Credentials
 	var err error
 	if c.ImpersonateServiceAccount != "" && !initialCredentialsOnly {
-		opts := option.ImpersonateCredentials(c.ImpersonateServiceAccount, c.ImpersonateServiceAccountDelegates...)
-		creds, err = transport.Creds(context.TODO(), opts, option.WithScopes(clientScopes...))
+		jsonCreds, err := credentials.DetectDefault(&credentials.DetectOptions{
```

Comment:
can we rename this to adcCreds or defaultCreds? There is no longer a direct JSON file so I think it will be less confusing.

Reply:
I guess there IS a JSON file, but we aren't supplying it like we did with CredentialsJSON option